specification/databox/resource-manager/readme.ruby.md

```diff
@@ -78,4 +79,14 @@ Please also specify `--ruby-sdks-folder=<path to the root directory of your azur
 ``` yaml $(tag) == 'package-2021-05' && $(ruby)
 namespace: "Azure::Compute::Mgmt::V2021_05_01"
 output-folder: $(ruby-sdks-folder)/management/azure_mgmt_databox/lib
+```
+
+### Tag: package-2021-12 and ruby
+
+These settings apply only when `--tag=package-2021-12 --ruby` is specified on the command line.
+Please also specify `--ruby-sdks-folder=<path to the root directory of your azure-sdk-for-ruby clone>`.
+
+``` yaml $(tag) == 'package-2021-12' && $(ruby)
+namespace: "Azure::Compute::Mgmt::V2021_12_01"
+output-folder: $(ruby-sdks-folder)/management/azure_mgmt_databox/lib
```

Comment:
I don't think we support ruby anymore. do you still need this ? if no, can you remove the configuration of SDK automation here https://github.com/Azure/azure-rest-api-specs/pull/16989/files#diff-2862cc253c3974820503835a06534fff6ae1a9f9dad08afa3c0a0ebac0231c81R122-R124

Reply:
I have removed "specification/databox/resource-manager/readme.ruby.md", thanks